README.md

```diff
@@ -64,8 +64,35 @@ The following is the schema of every collection in the database
     "createdAt" : ISODate("2018-03-10T07:17:53.741Z")
 }
 ```
+
+##### Channel
+```
+{
+    "_id" : ObjectId("5aa40a9e383fdf132025ccc4"),
+    "name" : "Test",
+    "users" : [
+        {
+            "userId" : ObjectId("5aa16bcedc440749402697ba"),
+            "acceptedInvite" : true
+        }
+    ],
+    "admin" : ObjectId("5aa16bcedc440749402697ba"),
+    "createdAt" : ISODate("2018-03-10T16:41:02.740Z")
+}
+```
+##### Message
+```
+{
+    "_id" : ObjectId("5aa54cae956974424063e266"),
+    "channelId" : ObjectId("5aa40a9e383fdf132025ccc4"),
+    "userName" : "5a9cb7dbc227cc57bc7b9555",
```

Comment:
This needs to be `userId` and it needs to be an ObjectId and not a string.

This means on creation you need to use `convertToObjectId` and when comparing you need to use `id1.equals(id2)` not `id1 === id2`. The equality operator does not work on `ObjectIds`